stage0/src/library/compiler/ir_interpreter.cpp

```diff
@@ -904,7 +904,7 @@ class interpreter {
     }
 public:
     explicit interpreter(environment const & env, options const & opts) : m_env(env), m_opts(opts) {
-        m_prefer_native = opts.get_bool(*g_interpreter_prefer_native, LEAN_DEFAULT_INTERPRETER_PREFER_NATIVE);
+        m_prefer_native = true; //opts.get_bool(*g_interpreter_prefer_native, LEAN_DEFAULT_INTERPRETER_PREFER_NATIVE);
     }
```

Comment:
Is this a temporary hack?

Reply:
Hah, you found it by yourself :grin: ! Yes, this is necessary because the `SourceInfo` change breaks the parser ABI (syntax objects created by the previous stage and consumed by macros in the current stage), so like we discussed before we have to make sure we do not run code expecting the new `SourceInfo` structure. The nice thing about changing it like this is that the change automatically vanishes with the next update-stage0.